cmd/ipfs/daemon.go

```diff
@@ -629,19 +629,50 @@ func mountFuse(req *cmds.Request, cctx *oldcmds.Context) error {
 }
 
 func maybeRunGC(req *cmds.Request, node *core.IpfsNode) (<-chan error, error) {
-	enableGC, _ := req.Options[enableGCKwd].(bool)
-	if !enableGC {
+	gc, err := initGC(req, node)
+	if err != nil {
+		return nil, err
+	}
+	if !gc.EnableGC {
+		log.Warningf("GC: %s", "Repo GC is disabled.")
```

Comment:
Why do we need this warning? Is it part of the original intention of the PR?

Reply:
> Is it part of the original intention of the PR?
We should print some warning message when the gc is disable.
You can see the comment in the [issue](https://github.com/ipfs/go-ipfs/issues/5140#issue-333793980)